fsd_utils/logging/logging.py

```diff
@@ -74,26 +74,31 @@ def before_request():
 
     @app.after_request
     def after_request(response):
-        current_app.logger.log(
-            logging.ERROR
-            if response.status_code // 100 == 5
-            else logging.INFO,
-            "{method} {url} {status}",
-            extra={
-                "status": response.status_code,
-                "duration_real": (
-                    (time.perf_counter() - request.before_request_real_time)
-                    if hasattr(request, "before_request_real_time")
-                    else None
-                ),
-                "duration_process": (
-                    (time.process_time() - request.before_request_process_time)
-                    if hasattr(request, "before_request_process_time")
-                    else None
-                ),
-                **_common_request_extra_log_context(),
-            },
-        )
+        if request.path != "/healthcheck":
+            current_app.logger.log(
+                logging.INFO,
+                "{method} {url} {status}",
```

Comment:
I'm wondering if this may get printed as that extra error if the response is null? It came originally from https://github.com/Crown-Commercial-Service/digitalmarketplace-utils/blob/main/dmutils/logging.py#L71 so they may be worth referring to if it's not that.

Reply:
I think it shows the string because the `extra` param evaluates to None perhaps?